subprojects/docs/src/docs/release/notes.md

```diff
@@ -55,9 +55,50 @@ Example:
 ADD RELEASE FEATURES BELOW
 vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv
 
+--> 
+<a name="http-build-cache-improvements"></a>
+## HTTP build cache usage improvements
+
+### Following redirects
+
+The built-in HTTP build cache connector now follows redirect responses, supporting a wider range of cache implementations and network management strategies.
+This can be leveraged to gracefully migrate to new cache locations, or support HTTP caches that redirect as a matter of course such as those that
+utilize some form of request signing to access storage systems.
+Following of redirects happens by default with no additional configuration needed.
+
+For more information on the effect of different types of redirects, see the documentation for [HttpBuildCache](dsl/org.gradle.caching.http.HttpBuildCache.html).
```

Comment:
Is this something that should be documented in the user guide, too?

Reply:
I don't think so. That is, I don't see the benefit in duplicating the information in these two places.

Reply:
I went ahead and duplicated.